.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -11,7 +11,26 @@ labels: 'type:bug, status:requirements, priority-5-triage'
 
 **What Renovate type, platform and version are you using?**
 
-<!-- Tell us if you're using the hosted App, or if you are self-hosted Renovate yourself. Platform too (GitHub, GitLab, etc) plus which version of Renovate if you're self-hosted. -->
+<!-- Tell us if you're using the hosted App, or if you are self-hosting Renovate yourself. Platform too (GitHub, GitLab, etc) plus which version of Renovate if you're self-hosted. -->
+
+**Renovate type**
+
+- [ ] Hosted app
+- [ ] Self hosted
+
+**Renovate platform**
+
+- [ ] Azure DevOps
+- [ ] Azure DevOps Server (self-hosted)
+- [ ] Bitbucket Cloud
+- [ ] Bitbucket Server
```

Comment:
```suggestion
- [ ] Bitbucket Cloud (bitbucket.org)
- [ ] Bitbucket Server (self-hosted)
```

I've opened a PR (#9867) to change the https://docs.renovatebot.com/modules/platform/ source file.
This suggestion has those exact same changes.